.chloggen/1016.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: enhancement
+component: db, messaging, gen_ai
```

Comment:
does this work? (asking for future me)

Reply:
should be fine, based on @joaopgrassi experiment in https://github.com/open-telemetry/semantic-conventions/pull/698#discussion_r1496194623 it should render as

- `messaging, db, gen_ai`: <title> (#PR)